docs/source/use_cases/discover_public_data/1000_genomes.rst

```diff
@@ -54,9 +54,9 @@ Google Cloud Platform data locations
 
 * Google Cloud Storage folders
    * These files were loaded into Google Genomics datasets:
-      * `gs://genomics-public-data/1000-genomes <https://console.developers.google.com/storage/genomics-public-data/1000-genomes/>`_
-      * `gs://genomics-public-data/1000-genomes-phase-3 <https://console.developers.google.com/storage/genomics-public-data/1000-genomes-phase-3/>`_
-   * A full mirror of http://ftp-trace.ncbi.nih.gov/1000genomes/ftp/ `gs://genomics-public-data/ftp-trace.ncbi.nih.gov/1000genomes/ftp/ <https://console.developers.google.com/storage/browser/genomics-public-data/ftp-trace.ncbi.nih.gov/1000genomes/ftp/>`_.
+      * `gs://genomics-public-data/1000-genomes <https://console.cloud.google.com/storage/genomics-public-data/1000-genomes/>`_
+      * `gs://genomics-public-data/1000-genomes-phase-3 <https://console.cloud.google.com/storage/genomics-public-data/1000-genomes-phase-3/>`_
```

Comment:
Same thing as above for the phase-3 link here:

https://console.cloud.google.com/m/cloudstorage/b/genomics-public-data/o/1000-genomes-phase-3/SEE_ALSO_THE_FULL_MIRROR_OF_1%2C000_GENOMES